packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -77,6 +84,23 @@ const MarketPlacePage = () => {
     'error'
   );
 
+  const handleInputChange = (input) => {
+    setSearchQuery(input);
+    setSearchResults(searchIndex.search(input));
+  };
+
+  useEffect(() => {
+    if (isLoading) return;
```

Comment:
I'd prefer putting everything in a `if (!isLoading)` condition instead, because returning in an effect has a special meaning, which may be confusing here